client/src/partials/vouchers/records.js

```diff
@@ -0,0 +1,196 @@
+angular.module('bhima.controllers')
+.controller('VouchersRecordsController', VouchersRecordsController);
+
+// dependencies injection
+VouchersRecordsController.$inject = [
+  'VoucherService', '$translate', 'NotifyService',
+  'JournalFilteringService', 'uiGridGroupingConstants',
+  'uiGridConstants', 'ModalService', 'util'
+];
+
+/**
+ * Vouchers Records Controllers
+ * This controller is responsible for display all vouchers made with BHIMA through
+ * the simple voucher or complex voucher modules
+ */
+function VouchersRecordsController(Vouchers, $translate, Notify, Filtering, uiGridGroupingConstants, uiGridConstants, Modal, util) {
+  var vm = this;
+
+  /** gobal variables */
+  vm.filterEnabled = false;
+  vm.gridOptions = {};
+  vm.gridApi = {};
+
+  /** paths in the headercrumb */
+  vm.bcPaths = [
+    { label : 'TREE.FINANCE' },
+    { label : 'TREE.VOUCHER_RECORDS' }
+  ];
+
+  /** buttons in the headercrumb */
+  vm.bcButtons = [
+    { icon: 'fa fa-filter', label: $translate.instant('FORM.BUTTONS.FILTER'),
+      action: toggleFilter, color: 'btn-default'
+    },
+    { icon: 'fa fa-search', label: $translate.instant('FORM.LABELS.SEARCH'),
+      action: search, color: 'btn-default'
+    },
+    { icon: 'fa fa-print', label: $translate.instant('FORM.LABELS.PRINT'),
+      action: printList, color: 'btn-default'
+    }
+  ];
+
+  // init the filter service
+  var filtering  = new Filtering(vm.gridOptions);
+
+  // link to the receipt
+  var linkReceiptTemplate = '<div style="padding: 5px;">' +
```

Comment:
Could you move these into their own HTML files?  It would make reading the JS a lot easier.